test/openshift/e2e/parallel/1-042_validate_status_host/02-check_URL.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: |
+    route_URL=$(oc get routes -o jsonpath="{.items[*]['spec.host']}" --field-selector metadata.name=example-argocd-server -n $NAMESPACE)
+    status_URL=$(oc get argocd -o jsonpath="{.items[*]['status.host']}" -n $NAMESPACE)
+
+    echo -e "route_URL:${route_URL}\nstatus_URL:${status_URL}"
+
+    sleep 15
+    
+    route_URL=$(oc get routes -o jsonpath="{.items[*]['spec.host']}" --field-selector metadata.name=example-argocd-server -n $NAMESPACE)
+    status_URL=$(oc get argocd -o jsonpath="{.items[*]['status.host']}" -n $NAMESPACE)
+
+    echo -e "route_URL:${route_URL}\nstatus_URL:${status_URL}"
```

Comment:
Why are we retrieving route_url and status_url twice? Is it because it might not have been created at the first time? In that case, can we use loop?